SMLHelper/Handlers/CoordinatedSpawnsHandler.cs

```diff
@@ -220,6 +220,40 @@ public SpawnInfo(string classId, Vector3 spawnPosition, Vector3 rotation)
             spawnType = SpawnType.ClassId;
         }
 
+        [JsonConstructor]
+        internal SpawnInfo(string classId, TechType techType, Vector3 spawnPosition, Vector3 rotation)
```

Comment:
The `rotation` is stored as a `Quaternion`, so this appears to be a mistake.
```suggestion
        internal SpawnInfo(string classId, TechType techType, Vector3 spawnPosition, Quaternion rotation)
```